examples/chip-tool/commands/payload/CredentialResponseParser.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#include "CredentialResponseParser.h"
+#include <credentials/CertificationDeclaration.h>
+#include <credentials/DeviceAttestationConstructor.h>
+#include <string>
+
+using namespace ::chip;
+
+enum
+{
+    // Attestation Elements
+    kTag_CertDeclaration = 1,
+    kTag_Nonce           = 2,
+    kTag_TimeStamp       = 3,
+    kTag_FirmwareVersion = 4,
+};
+
+CHIP_ERROR AttestationElementsParser::Run()
```

Comment:
This can be done by installing a DeviceAttestationVerifier that does the logging you want and then wraps a DeviceAttestationVerifier obtained from `GetDefaultDACVerifier`.

The DeviceAttestationVerifier methods are always called by chip-tool's commissioner